oauth2_provider_jwt/views.py

```diff
@@ -16,6 +23,27 @@ class WrongUsername(Exception):
     pass
 
 
+class JWTAuthorizationView(views.AuthorizationView):
+
+    def get(self, request, *args, **kwargs):
+        response = super(JWTAuthorizationView, self).get(request, *args,
+                                                         **kwargs)
+        if request.GET.get('response_type', None) == 'token' \
+                and response.status_code == 302:
+            url = urlparse(response.url)
+            params = parse_qs(url.fragment)
+            content = {
+                'access_token': params['access_token'][0],
+                'expires_in': int(params['expires_in'][0]),
+                'scope': params['scope'][0]
+            }
+            jwt = TokenView()._get_access_token_jwt(request, content)
+            response = OAuth2ResponseRedirect(
+                response.url + '&access_token_jwt=' + jwt,
```

Comment:
More readable and safer is to use format: `'{}&access_token_jwt={}'.format(response.url, jwt)`